docs/architecture/_index.md

```diff
@@ -0,0 +1,188 @@
+- Multiple individual machines, linked by a {{< glossary_tooltip term_id="remote-part" text="remote" >}} connection.
+
+These two options are very similar; in both cases, the parts communicate with each other securely using gRPC/{{< glossary_tooltip term_id="webrtc" text="WebRTC" >}}.
+
+Because the parts are interconnected, you can write SDK code that establishes a connection with one of them and controls them all in a coordinated way.
+
+TODO: Find a good example, and explain why you'd want to use one multi-part machine versus many single-part machines
+
+{{< expand "Multi-part example" >}}
+
+Imagine you have a robotic arm with a camera on it, connected to a SBC such as a Raspberry Pi.
+You want to use the Viam motion service to control the arm, and you want to use the vision service on the output from the camera, but the SBC does not have the compute power to plan complex motion and also interpret camera output quickly.
+You can set up a second {{< glossary_tooltip term_id="part" text="part" >}} on a desktop or laptop computer, either as a sub-part of the same machine or as a separate machine connected as a remote, and offload the heavy compute to that part.
+
+{{< /expand >}}
+
+See [Parts, Sub-parts and Remotes](/architecture/parts/) for more details.
+
+## Communication flow
+
+TODO: Communication flow diagram
+
+On startup, `viam-server` establishes a {{< glossary_tooltip term_id="webrtc" text="WebRTC" >}} connection with the [Viam app](https://app.viam.com).
+`viam-server` pulls its configuration from the app, caches it locally, and initializes all components and services based on that configuration.
+
+If sub-parts or remote parts are configured, communications are established between the `viam-server` instances on each of them.
+
+If you have client code running on a separate computer, that code sends API requests to `viam-server` over wifi using gRPC.
+If a built-in service is communicating with a component, for example when the vision service requests an image from a camera, `viam-server` handles that request.
+
+{{% alert title="Protobuf APIs" color="info" %}}
+All Viam APIs are defined with the [Protocol Buffers (protobuf)](https://protobuf.dev/) framework.
+This is what enables Viam to provide SDKs in a variety of different programming languages.
+{{% /alert %}}
+
+When you control your machine or view its camera streams or sensor outputs from the Viam app **CONTROL** tab, those connections happen over WebRTC.
+The Viam app hits the same API endpoints as your SDK client code, with `viam-server` handling requests.
+
+{{% alert title="Security" color="info" %}}
+TLS certificates certificates provided by the app ensure that all communication is authenticated and encrypted.
+{{% /alert %}}
+
+For more details, see [Machine-to-Machine Communication](/architecture/machine-to-machine-comms/).
+
+## Data management flow
+
+TODO: Data flow diagram
```

Comment:
will send you a slack with a diagram bill used around this

Reply:
Maybe the simple machine diagram higher up covers it?

Reply:
Hmm. Maybe simplified would help? I think maybe you want to also bring into it a visual that shows the cache on the device (with a label cache). As well as query options?

Reply:
Hmm how would query options fit in? You mean like, add in a client workstation to the diagram that is requesting data from the cloud?

Reply:
![image](https://github.com/user-attachments/assets/c23af511-98a6-492d-8821-540f5b8ed60a)
might actually be good